src/Elcodi/Plugin/ProductCsv/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "elcodi-plugins/product-csv",
+    "license": "MIT",
+    "type": "elcodi-plugin",
+    "description": "Elcodi plugin to add Product import/export from .csv files",
+    "authors": [
+        {
+            "name": "Berny Cantos",
+            "email": "be@rny.cc"
+        },
+        {
+            "name": "Elcodi Community",
+            "homepage": "https://github.com/elcodi-plugin/product-csv/contributors"
+        }
+    ],
+    "require": {
+        "goodby/csv": "~1.0",
```

Comment:
5.4 is always required by all repositories.

And then, do we have some Symfony components namespaces in this repository

```php
use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
use Symfony\Bundle\FrameworkBundle\Controller\Controller;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\HttpFoundation\ResponseHeaderBag;
use Symfony\Component\HttpFoundation\StreamedResponse;
```

And services requirements:

``` php
$repository = $this->get('elcodi.repository.product');
```

So this means that all this dependencies must be defined in a hard way in the plugin.